app/Filament/Organizations/Resources/BeneficiaryResource/Pages/ViewInitialEvaluation.php

```diff
@@ -58,9 +59,14 @@ public function infolist(Infolist $infolist): Infolist
                                 ->schema(EditViolence::getInfoListSchema())]),
                     Tabs\Tab::make(__('beneficiary.wizard.risk_factors.label'))
                         ->schema([
-                            Section::make(fn ($record) => $record->riskFactors->risk_level->label() ??
-                                __('beneficiary.wizard.risk_factors.label'))
+                            Section::make()
                                 ->schema([
+                                    View::make('filament.notice')
+                                        ->viewData([
+                                            'message' => $this->record->riskFactors->risk_level->label(),
+                                            'bgClass' => sprintf('bg-%s bg-custom-200 text-custom-800', $this->record->riskFactors->risk_level->color()),
```

Comment:
`bg-%s` nu e corect. Clasele în tailwind trebuie să fie scrise complete, altfel nu sunt generate în build-ul de prod.

Prefer o altă abordare în loc de cum ai făcut `bgClass`:

```suggestion
                                            'bgColor' => $this->record->riskFactors->risk_level->color(),
```

_Continuarea în `notice.blade.php`_